calico_node/startup/startup.go

```diff
@@ -771,6 +771,11 @@ func ensureDefaultConfig(c *client.Client, node *api.Node) error {
 		return err
 	}
 
+	// Update the ClusterType with the type in CLUSTER_TYPE
```

Comment:
Yeah, good point.. maybe we leave the name singular to match then.... even though it's a list :/